source/guide_openslides.rst

```diff
@@ -0,0 +1,221 @@
+
+##########
+OpenSlides
+##########
+
+.. tag_list::
+
+OpenSlides is a free, web-based presentation and assembly system for managing and projecting agenda, motions, and elections of assemblies.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Python <lang-python>` and its package manager pip
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`Web Backends <web-backends>`
+
+License
+=======
+
+OpenSlides_ is released under the `MIT License`_.
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Install Python package
+----------------------
+Install the OpenSlides Python package:
+
+::
+
+ [isabell@stardust ~]$ pip3.9 install openslides --user
+ [...]
+ Running setup.py install for pyrsistent ... done
+ Running setup.py install for PyPDF2 ... done
+ Running setup.py install for roman ... done
+ Running setup.py install for websockets ... done
+ Running setup.py install for openslides ... done
+ [...]
+ [isabell@stardust ~]$
+
+Check if OpenSlides is installed by typing:
+
+::
+
+ [isabell@stardust ~]$ openslides --version
+ 3.3
+ [isabell@stardust ~]$
+
+Create configuration
+--------------------
+Run the following command to create the configuration:
+
+::
+
+ [isabell@stardust ~]$ openslides createsettings
+ Settings created at /home/isabell/.config/openslides/settings.py
+ [isabell@stardust ~]$ 
+
+Furthermore, populate the SQLite database by running the following command:
+
+::
+
+ [isabell@stardust ~]$ openslides migrate
+ Operations to perform:
+   Apply all migrations: agenda, assignments, auth, contenttypes, core, mediafiles, motions, sessions, topics, users
+ Running migrations:
+   Applying contenttypes.0001_initial... OK
+ [...]
+ [2021-04-11 20:59:34 +0200] [21317] [INFO] openslides.core.apps [zxnj] Updated config variables
+ [isabell@stardust ~]$
+
+ 
+Create web backend
+------------------
+
+.. note::
+
+    OpenSlides is running on port 8000.
+
+.. include:: includes/web-backend.rst
+
+
+Create service
+--------------
+
+You should set up a service that keeps OpenSlides alive while you are gone. We will use ``daphne`` as an ASGI backend server for the OpenSlides application. Create the file ``~/etc/services.d/openslides.ini`` with the following content:
+
+.. code-block:: ini
+
+ [program:openslides]
+ command=daphne -b 0.0.0.0 -p 8000 openslides.asgi:application
+ autostart=true
+ autorestart=true
+ stopsignal=INT
+
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration.
+
+
+Access OpenSlides
+-----------------
+Now point your Browser to your installation URL ``https://isabell.uber.space``.
+
+Use ``admin`` as username and ``admin`` as password for your first login. You should change this password at ``https://isabell.uber.space/users/password`` immediately after login!
```

Comment:
The credentials do not work on my test instance and I couldn't find any documentation on the default PW. Can you point me to where you got this from?

Reply:
The default credentials are shown on the login page after the installation.
You will find here also, that the default is "admin" and "admin": https://docs.openslides.org/OpenSlides-Handbuch-2.1b1.pdf

Reply:
Just got the message here on a fresh uberspace: 
![openslides_login_defaultcreds](https://user-images.githubusercontent.com/3757157/114573887-13acce00-9c79-11eb-8423-640f78d20d2a.PNG)

Reply:
worked fine on the 2nd try :shrug: thanks!